internal/apiclient/dashboard.go

```diff
@@ -126,6 +126,10 @@ func (c *DashboardAPIClient) AddRun(ctx *config.RunContext, out output.Root) (Ad
 			addRun(run: $run) {
 				id
 				shareUrl
+				organization {
+					id
+					name
+				}
```

Comment:
CI fails because production dashboard API doesn't have this field yet. Ran it locally with `INFRACOST_DASHBOARD_API_ENDPOINT` and it was green.

Not sure how to approach this and I question if we should do real requests to API. Yes, it's closer to the actual use, but API CI should cover this case already, we just need to test that the success/failure response is handled correctly which can be emulated with stubbed requests. Making real requests also pollutes production db as CI is run pretty frequently.